sway-core/src/error.rs

```diff
@@ -995,6 +995,10 @@ pub enum CompileError {
     NonConstantDeclValue { span: Span },
     #[error("Declaring storage in a {program_kind} is not allowed.")]
     StorageDeclarationInNonContract { program_kind: String, span: Span },
+    #[error("\"break\" used outside of a loop")]
```

Comment:
I've made a comment below in the formatter regarding the `unimplemented!()` macro. Is this a warning / error that would be displayed in that section or is this error used somewhere else?

Reply:
This is a regular compiler error. This is the desirable behavior when `break` is used outside a loop. This is exactly what Rust tells you when you do that.